jobs/mysql_restore_job.go

```diff
@@ -35,9 +35,10 @@ import (
 
 const (
 	xbstreamBin      = "xbstream"
-	mySQLServiceName = "mysql"
+	mySQLServiceName = "mysqld"
```

Comment:
I wonder how it worked before? 🤔
Why we faced this bug only during first restore?

Reply:
Briefly, due to we don't stop the service (because we check the wrong one) file /var/lib/mysql/mysql.sock is not recreated in correct place. So, the restoring being processed correctly, we just cannot connect the DB. On the second try it works other way because we stop the service (stopping mysql causes stopping mysqld).

Reply:
Got it, thanks!